### PR TITLE
Fix case-sensitive spelling in scripts that get deployed to C++ repo

### PR DESCRIPTION
**Pull request by antkmsft**
For C++-only fixes, see https://github.com/Azure/azure-sdk-for-cpp/pull/5346

### CODE REVIEW

File: eng/scripts/Set-VcpkgWriteModeCache.ps1
Comment (danieljurek) [inline eng/scripts/Set-VcpkgWriteModeCache.ps1:1]: But there is no `azblob` in this file.

Comment (danieljurek) [reply eng/scripts/Set-VcpkgWriteModeCache.ps1]: Nevermind... I somehow didn't expand the file all the way down to the last couple lines :P

**Review by danieljurek — APPROVED** [review]

**Review by benbp — APPROVED** [review]

**Comment by antkmsft** [comment]
Please let me know if in the future you would want me to self-approve "create sync PRs" CI step after I get your approvals, or if you typically prefer to do it.

**Pull request merged**
